data_steward/validation/sql_wrangle.py

```diff
@@ -9,22 +9,32 @@
 TEMP_TABLE_PATTERN = re.compile('\s*INTO\s+([^\s]+)')
 TRUNCATE_TABLE_PATTERN = re.compile('\s*truncate\s+table\s+([^\s]+)')
 DROP_TABLE_PATTERN = re.compile('\s*drop\s+table\s+([^\s]+)')
-
+COMMENTED_BLOCK_REGEX = re.compile('(?P<before_comment>(^)(.)*)(?P<comment>(\/\*)(.)*(\*\/))(?P<after_comment>(.)*$)', re.DOTALL)
```

Comment:
Is this a bad merge perhaps? These changes don't look related and seem like they may be from one of Nishanth's recent PRs.

Reply:
This actually happened.  This module was being tested from main_test.py instead of its own sql_wrangle.py.  So the sql_wrangle tests were removed from main_test.py and put into a created sql_wrangle_test.py module.  In doing this, I wrote a unit test that broke the `is_commented_block()` function.  The previous function couldn't handle true block comments.  This passed as an SQL block comment:
```
-- Some text

-- More text
```

This failed:
```
/*
A block comment

Continuing.
*/
```
So I wrote this to allow the unit test to pass.